<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OutputTab</class>
 <widget class="QWidget" name="OutputTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>392</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QPushButton" name="butBrowseOutput">
     <property name="text">
      <string>Browse...</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLineEdit" name="inputOutputFolder">
     <property name="placeholderText">
      <string>Output folder</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_14">
      <item row="0" column="0">
       <widget class="QCheckBox" name="resize">
        <property name="text">
         <string>Resize</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="inputResize"/>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="rename">
        <property name="text">
         <string>Rename</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="inputRename"/>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="watermark">
        <property name="text">
         <string>Watermark</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="inputWatermark"/>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="butBrowseWatermark">
        <property name="text">
         <string>Browse...</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="comment">
        <property name="text">
         <string>Comment</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="inputComment"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
